com.unity.shadergraph/Editor/Importers/ShaderGraphImporter.cs

```diff
@@ -123,53 +182,16 @@ public override void OnImportAsset(AssetImportContext ctx)
             graph.OnEnable();
             graph.ValidateGraph();
 
-            Shader shader = null;
+            // Shader shader = null;
+            UnityEngine.Object mainObject = null;
```

Comment:
nit: any reason to declare this on it's own line instead as the return for `BuildAllShaders`?

Reply:
because of the if () around the assignment.

Reply:
oh...the #if threw me off and I thought it was just a scope